2017-01-30-openwhisk-integration-with-serverless

```diff
@@ -0,0 +1,44 @@
+---
+title: Introducing OpenWhisk Integration with the Serverless Framework
+description: The new OpenWhisk provider plugin allows developers to build applications for the platform using the Serverless Framework.
+date: 2017-01-30
+thumbnail: https://cloud.githubusercontent.com/assets/20538501/22410455/110d1f36-e65e-11e6-8db8-87e834504e13.jpg
+layout: Post
+---
+Today we're excited to announce [OpenWhisk](https://developer.ibm.com/openwhisk/) integration with the Serverless Framework!
+
+The official [OpenWhisk Provider Plugin](https://github.com/serverless/serverless-openwhisk) allows developers to build applications for the platform using the Serverless Framework.
```

Comment:
"For the Bluemix platform"? (Not sure if it's a platform, but it's their name for the Cloud where OpenWhisk runs on.)